sycl/doc/extensions/experimental/sycl_ext_intel_fpga_task_sequence.asciidoc

```diff
@@ -0,0 +1,307 @@
+
+== `task_sequence` Properties
+
+The following code and table describe the "balanced", "invocation_capacity", and "response_capacity" properties that can be provided when declaring a `task_sequence`.
+
+[source,c++,linenums]
+----
+namespace sycl {
+namespace ext {
+namespace intel {
+namespace experimental {
+
+struct balanced {
+  using value_t = property_value<balanced>;
+};
+
+struct invocation_capacity {
+  template <uint32_t Size>
+  using value_t = property_value<invocation_capacity, std::integral_constant<uint32_t, Size>>;
+};
+
+struct response_capacity {
+  template <uint32_t Size>
+  using value_t = property_value<response_capacity, std::integral_constant<uint32_t, Size>>;
+};
+
+inline constexpr balanced::value_t balanced_v;
+
+template <uint32_t Size>
+inline constexpr invocation_capacity::value_t<Size> invocation_capacity_v;
+
+template <uint32_t Size>
+inline constexpr response_capacity::value_t<Size> response_capacity_v;
+
+} // namespace experimental
+} // namespace intel
+} // namespace ext
+} // namespace sycl
+----
+
+--
+[options="header"]
+|===
+| Property | Description
+| balanced | The `balanced` property is a guarantee to the SYCL device compiler that the `task_sequence` object will call exactly the same number of `async` s and `get` s over the object's lifetime (i.e. before the `task_sequence` destructor is invoked). 
+
+| invocation_capacity | The `invocation_capacity` property defines the minimum required number `Size` of `async` invocations that must be invokable without any `get` being called on that same set.  This number of `async` invocations without a `get` call is the minimum number that will be supported before the `async` member function blocks.
+
+| response_capacity | The `response_capacity` property tells the compiler the required simultaneous capacity `Size` of get calls. The compiler must guarantee storage capacity for results of up to `Size` `async` calls.
```

Comment:
Can you rephrase this to be user-centric like the definition of invocation capacity rather than compiler-centric?

E.g. with a response capacity of X the user can be sure up to X outstanding calls can make progress but if there are more than X the X+1th may block until at least one more get() is called.

Reply:
Updated. Please take a look.